spec.bs

```diff
@@ -282,6 +297,16 @@ navigator.credentials.create({ publicKey })
 
 </div> <!-- non-normative -->
 
+**TODO**: Sugest we provide clarifying text/expand around the following 
+* Does the extension imply that this credential my ONLY be used for Payment, or does this mean the WebAuthn credential could be used for Login or Payment, 
+  and both are legitimate?
+* Can the payment extension be 'added' once the webAuthn is created? Or must it be set at registration time? and can it be disabled too? 
```

Comment:
Currently no, but I think we should think about this. As noted, a lot here depends on the WebAuthn WG's preference/requirements on whether SPC credential == WebAuthn credential, or if a separate bit (of some sort) must be set.

I *think* in general that WebAuthn doesn't allow modifying credentials (maybe I am incorrect), and so its generally a replace operation for something like an 'upgrade'.

Reply:
That's my understanding as well. Let's state this is the situation right now. (Do you think we need to verify this with WebAuthn team?)

Reply:
I will be speaking at the WebAuthn WG tomorrow afternoon EST, and I expect a resolution to come out of that.